mix.exs

```diff
@@ -12,26 +12,14 @@ defmodule ExometerZabbix.Mixfile do
      deps: deps()]
   end
 
-  # Configuration for the OTP application
-  #
-  # Type "mix help compile.app" for more information
   def application do
-    [applications: [:logger, :exometer_core]]
+    [extra_applications: [:logger]]
   end
 
-  # Dependencies can be Hex packages:
-  #
-  #   {:mydep, "~> 0.3.0"}
-  #
-  # Or git/path repositories:
-  #
-  #   {:mydep, git: "https://github.com/elixir-lang/mydep.git", tag: "0.1.0"}
-  #
-  # Type "mix help deps" for more examples and options
   defp deps do
     [
       {:exometer_core, "~> 1.5" },
-      {:poison, "~> 2.0 or ~> 3.0"},
+      {:jason, "~> 1.0", runtime: false},
```

Comment:
Why does it need `runtime: false`. I believe Jason doesn't start an application so it wouldn't be necessary?

Reply:
It's not necessarily needed, as indeed Jason will not start anything. However, I like it because it's more specific. Also, by omitting `runtime: false`, it will still be added in the `exometer_zabbix.app` file to the applications, so it will try to start Jason.
But indeed, it's not necessary.

Reply:
I like it. It's cleaner indeed.